fix: Instructions to build & Update dependencies to work properly (#6)
revoltchat/mutiny: README.md

GTK4 Revolt client

## Installation

> The following components are required beforehand:

Reviewer: ```suggestion
First, make sure the following components are installed:
```

* (The meson build system)[https://mesonbuild.com/Quick-guide.html]
* (The Rust toolchain)[https://www.rust-lang.org/tools/install]
* (The GTK4 library)[https://github.com/ToshioCP/Gtk4-tutorial/blob/main/gfm/sec2.md]
* (The Adwaita library)[https://github.com/GNOME/libadwaita]

> It's then essential to setup the build directory with meson:

Reviewer: ```suggestion
After installing the necessary components, set up `builddir`:
```

```sh
meson setup builddir
```

> Once this is succesful; It's then proper to perform the compilation procedure:

Reviewer: ```suggestion
Once this is done, compile the app:
```

```sh
cd builddir && meson compile
```

Unfortunely, in some cases it is difficult to utilize `meson install` properly. In this case the user can just specify the prefix to the current build directory:
```sh
# inside builddir

> meson configure -D `pwd`

Reviewer 2: I'm not sure what this is supposed to do. The `-D` command-line flag. expects to receive an argument such as `option_name=value`, and you're giving it the current directory which is a value without an option name.

Guessing based on the context of "the `share` folder on the build directory" that this is supposed to be `-D prefix=$(pwd)`?

Reviewer 2: It seems to me like you probably don't want to do any of this. Instead I would investigate https://mesonbuild.com/Commands.html#devenv which you can also use in combination with `meson.add_devenv()` to enhance the devenv shell with e.g. XDG_DATA_DIRS, use gnome.compile_schemas(), and so on.

> meson configure -D`pwd`

Reviewer: I assume this needs a space?
```suggestion
meson configure -D `pwd`
```

```

When this is done, the resources will be located inside the `share` folder on the build directory.

> If not installing the program with `meson install` then it is nescesary to notify glib of the schemas residing inside the `data/` folder; so the client shall be run as:

Reviewer: ```suggestion
If the app is not installed with `meson install`, you'll need to tell glib that the schemas reside inside the `data/` folder; therefore, run the client with this command:
```
